rancher/v1.6/en/cattle/volumes/index.md

```diff
@@ -0,0 +1,232 @@
+
+```
+$ rancher volume ls
+```
+
+If you created the `app-data` volume from the previous section, you may wonder why it wasn't listed. You can include `inactive` volumes with the `--all` or `-a` flag.
+
+```
+$ rancher volume ls --all
+```
+
+#### Delete a Volume
+
+You can delete a volume with the `rancher volume rm` command.
+
+```
+$ rancher volume rm app-data
+```
+
+#### Volume States
+
+Volumes have seven distinct states: `inactive`, `activating`, `active`, `deactivating`, `detached`, `removing` and `removed`.
+
+A newly-created volume is in `inactive` state until a container attempts to attach to it.
+
+When a container is created, associated volumes enter `activating` state. Once the container enters the `running` phase, its volumes enter `active` state. Containers attaching to an already `active` volume will not impact that volume's state.
+
+When all containers attached to a volume are marked for deletion, the volume enters a `dactivating` state. Once the conatiners are deleted, the volume enters `detached` state.
+
+When a volume is marked for deletion, it enters a `removing` state. Once the data is deleted from the host(s), it enters `removed` state. Removed volumes will not appear when listing volumes, but they continue to exist in Rancher API for a period of time for debugging and auditing purposes.
+
+### Volume Scopes
+
+There are two different volume scopes. The scope refers to the level at which the volume is managed by Rancher.
+
+#### Environment Scope
+
+An `environment` scope volume may be shared by all containers in an environment. Rancher schedules a container appropriately based on the hosts that have access to its assigned volumes.
+
+Environment scope volumes are not automatically shared across all hosts in an environment. You need to be using a shared volume driver such as `rancher-nfs` to accomplish that. This means that an `environment` scope volume using the `local` driver will only be available on a single host and any container using that volume will be scheduled to that host.
+
+Rancher requires that an environment volume exists prior to creating a service that consumes it. Any installed volume driver can be used.
+
+The practical benefit of this scope is the ability to easily share data between different software services/stacks whose lifecycles are independently managed. The user is given full control and must [manage the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#managing-volumes) themselves.
+
+##### Shared Environment Scope Volume Example
+
+First, [create the environment scope volume]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#create-a-volume) to be shared between stacks.
+
+```
+$ rancher volume create --driver local redis-data-external
+```
+
+Define the volume with `external: true` modifier to indicate the volume's lifecycle is managed externally (environment scope). Save the `docker-compose.yml` file below to a directory.
+
```

Comment:
make a note that without `external:true`, the volume will be stack scoped and refer to the stack scoped section